tools/astarte_dev_tool/lib/commands/dashboard/open.ex

```diff
@@ -0,0 +1,85 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2024 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule AstarteDevTool.Commands.Dashboard.Open do
+  @moduledoc false
+
+  require Logger
+
+  def exec(opts) do
+    dashboard_url = Keyword.get(opts, :dashboard_url, "http://dashboard.astarte.localhost/")
```

Comment:
like realm_name and realm_private_key, docs should also state the possibility of dashboard_url (and the default value)

Reply:
The options was already documented between the command options, but I expanded the example in the README for reporting all existing options there too.